exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -126,14 +128,23 @@ private float getHeapUsage() {
   public static class QueryResult {
     private final String queryId;
     public final Collection<String> columns;
+    public final HashMap<String, ArrayList> metadata;
```

Comment:
Use more generic collections: `public final Map<String, List> metadata;`.
Plus List of what type? Maybe `List<String>`?

Please update here and in the code below.